build-images.sh

```diff
@@ -33,6 +41,8 @@ main() {
   readarray -t assets < <(
     build_kexec_bundle "$tag" "$arch" "$tmp"
     build_netboot_image "$tag" "$arch" "$tmp"
+    build_gnome_netboot_image "$tag" "$arch" "$tmp"
+    build_plasma5_netboot_image "$tag" "$arch" "$tmp"
```

Comment:
We need to check if those images can be still uploaded to github / fit in RAM.

Reply:
They're about the same size as the corresponding ISOs (~2GiB for GNOME, 1.4GiB for Plasma 5). I believe that should still be fine for both GitHub and RAM :smile:

Reply:
Ok. I am wondering why our normal kexec image is than so big (1.3GB)...

Reply:
Looks like something changed between `20.09` and `21.05` where `20.09` is only 600MiB and `21.05` is 1.3GiB for the `kexec-bundle`